plugins/Deflect/DeflectPlugin.cpp

```diff
@@ -246,14 +266,8 @@ class DeflectPlugin::Impl
             case deflect::Event::EVT_VIEW_SIZE_CHANGED:
             {
                 Vector2ui newSize(event.dx, event.dy);
-                if (_engine.getCamera().isSideBySideStereo())
-                    newSize.x() *= 2;
-
                 if (_params.getResizing())
-                {
-                    _appParams.setWindowSize(
-                        _engine.getSupportedFrameSize(newSize));
-                }
+                    _appParams.setWindowSize(newSize);
```

Comment:
I think this is not correct. In braynsViewer (not sure about the service), the window size is the union of the frame buffers and that's what you set in the application parameters, but here the size is supposed to be the size of just a buffer. Does it mean the stereo streaming from braynsService is not supported?

I tried to see what I get in this case myself, but as a matter of fact there seems to be another problem when streaming from braynsViewer as the brayns process window is ping-pong'ing  between two different sizes, even without enabling stereo. On the other hand, braynsService is OK.